scripts/mint.ts

```diff
@@ -1,11 +1,13 @@
 import {
   SigningCosmWasmClient,
   MsgExecuteContractEncodeObject,
-} from '@cosmjs/cosmwasm-stargate';
-import { DirectSecp256k1HdWallet } from '@cosmjs/proto-signing';
-import { coins, calculateFee, GasPrice } from '@cosmjs/stargate';
+  DirectSecp256k1HdWallet,
+  coins,
+  calculateFee,
+  GasPrice,
+  toUtf8,
+} from 'cosmwasm';
 import { MsgExecuteContract } from 'cosmjs-types/cosmwasm/wasm/v1/tx';
```

Comment:
Independent of this PR: it seems like `cosmjs-types` is missing in the dependencies list

Reply:
Ideally `MsgExecuteContract` is re-exported from `cosmwasm`, such that you don't have to worry about the extra dependency.

Reply:
Yeah, this is how I was expecting it work with CosmJS previously too but figured the library wanted to expose an abstraction over batching eventually and that's why it was excluded.

Reply:
Ah, so you need a remake of https://github.com/cosmos/cosmjs/pull/876? Happy to add. The API in the PR was just not correct enough to get in.

Reply:
Tracked in https://github.com/cosmos/cosmjs/issues/1072 now